hamid914-shahrekord.md

```diff
@@ -0,0 +1,5 @@
+## Shahrekord, Chaharmahal-O-Bakhtiari
```

Comment:
Shahrekord city is located in Chahrmahal-O-Bakhtiari state that is located in Iran.
[Shahr-E Kord](https://en.wikipedia.org/wiki/Shahr-e_Kord)